### DISCUSSION TITLE
Sandbox environment URL in the console

### DISCUSSION THREAD

**Discussion by poshgrad** [Azure/api-management-developer-portal#2558] [Feature request]
Good morning, 

We have developed an API to push orders to our ERP backend. It is in production. 
We can"t however allow a working TRY IT to the developers so that they can explore the API on their own since the call would end up in our Production backend.
I would like to redirect the calls done with our TRY IT to our Staging configuration but it seems not to be possible to replace the Request URL in the TEST tab by the one of our staging environment (the area is greyed). 
It prevents de facto the developers to use the TRY IT in the API dev portal in production and makes it useless. 
Would it be possible to be able to configure the URL and authentication behind the TRY IT so that we could redirect the calls to our Staging environment?

BR

Julien

**Comment by mikebudzynski** [comment]
@poshgrad Thanks for requesting the feature. We do have plans to make changes in this area in 2021 and 2022, although at this moment I can't confirm if they will solve your problem. Meanwhile, you can use the self-hosted portal version and replace the URL in the interactive console on the client-side of the portal.

**Comment by AdamCoulterOz** [comment]
@mikebudzynski any updates on this?
